lib/cog/queries/relay_group_queries.ex

```diff
@@ -14,10 +15,24 @@ defmodule Cog.Queries.RelayGroup do
     |> where([rg], rg.id == ^id)
   end
 
+  def add!(:bundle, group_id, bundle_id),
```

Comment:
I think we're mixing concerns in this module now.

These query modules should contain functions that generate Ecto queries (i.e., create data structures), not functions that actually _execute_ the queries. Ecto queries have the nice quality of composability, so we can provide query "building blocks" here that we can use elsewhere in various combinations. Having functions that also execute queries as a side effect destroys this composability.

These functions actually execute operations in the database, and so are actually defining the actual behavioral API we want to expose around bundles and their groups.

I've been meaning to write up something on shifting toward a more "classic DDD repository" approach for how we interact with the system, where we expose the actual behavioral API in a module / series of modules, and _that_ is how we manipulate the models in our domain. Basically, all database queries and `Cog.Repo` access occur behind that API. This encapsulates the behavior very clearly, ensures consistency, allows us more leeway in testing (simply substitute something else that implements that interface), and allows us to introduce things like caching transparently (as opposed to our past/current practice of calling caches directly in code).

We don't need to adopt this approach right now in this PR, but I would recommend at least that we keep actual database manipulation calls out of these query modules.